src/service/poc_lora.proto

```diff
@@ -0,0 +1,91 @@
+syntax = "proto3";
+
+package helium.poc_lora;
+
+import "data_rate.proto";
+
+enum invalid_participant_type {
+  beaconer = 0;
+  witness = 1;
+}
+
+enum invalid_reason {
+  none = 0;
+  bad_signature = 1;
+  not_asserted = 2;
+  entropy_expired = 3;
+  bad_entropy = 4;
+  invalid_capability = 5;
+  invalid_pubkey = 6;
+  irregular_interval = 7;
+  // TODO: add final values
+}
+
+message lora_ingest_report_v1 {
+  uint64 received_timestamp = 1;
+  oneof report {
+    lora_beacon_report_req_v1 beacon = 2;
+    lora_witness_report_req_v1 witness = 3;
+  }
+}
+
+message lora_beacon_report_req_v1 {
```

Comment:
Why is there a `req` in the name here?

Reply:
the 'req' protos are the incoming grpc protos....ie the gateway rs client submits requests of type: lora_beacon_report_req_v1 and lora_witness_report_req_v1.  The ingestor writes out those req protos.

The verifier then consumes those protos and writes them out as part of its valid and invalid protos.  The one/same proto is used as there is an existing encoder/decoder capability.

The alternative is to have a new proto but you will end up with the same structure and thus duplication